packages/apputils/package.json

```diff
@@ -42,12 +42,12 @@
     "@phosphor/signaling": "^1.2.2",
     "@phosphor/virtualdom": "^1.1.2",
     "@phosphor/widgets": "^1.5.0",
+    "@types/react": "~16.0.19",
```

Comment:
What about making it `^16.0.9`? Is that backwards incompatible? I notice that we ship with react 16.2, IIRC.

Reply:
I'm not sure. I experimented with a bunch of different semver combinations in #4367, and couldn't really find something that makes much of a difference. I don't see a harm to it, though.

Reply:
at least it opens up the possible typings, assuming that the typing *we* use in this package is compatible across these versions.

Reply:
That is where I noticed a real difference between `npm` and `yarn`: `npm` was correctly deduplicating this dependencie according to the semver range, and `yarn` was not.

Reply:
Just changed it to `^`.

Reply:
Just changed it back...sorry for the noise.

Reply:
> That is where I noticed a real difference between npm and yarn: npm was correctly deduplicating this dependencie according to the semver range, and yarn was not.

*Very* interesting!